src/components/projects-summary/projects-summary.html

```diff
@@ -99,4 +99,25 @@ <h2 class="secondary" ng-if="$ctrl.projects && $ctrl.projects.length > 0">Gettin
       </ul>
     </div>
   </div>
+  <div ng-if="$ctrl.recentlyViewedItems.length">
+    <h2 class="secondary">Recently Viewed</h2>
+    <div class="services-view">
+      <div class="services-view-container">
+        <div class="services-items-inner">
+          <div class="services-items-inner-shadow-covers">
+            <a href="" class="services-item" ng-repeat="item in $ctrl.recentlyViewedItems track by (item.resource | uid)"
```

Comment:
Ok, that is a design question best answered by @beanh66 and @serenamarie125.  I think the design specifically wanted the most recently clicked item to appear left-most in the list.  Pretty sure I specifically asked @beanh66 that exact question.